Src/BootCamp.Chapter/Computer/Motherboard.cs

```diff
@@ -2,5 +2,11 @@
 {
     public class Motherboard
     {
+        private string _motherboard;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_motherboard' or refactor the code to use its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)